COPYING

```diff
@@ -7,8 +7,8 @@
  of this license document, but changing it is not allowed.
 
 [This is the first released version of the Lesser GPL.  It also counts
- as the successor of the GNU Library Public License, version 2, hence
- the version number 2.1.]
+ as the successor of the GNU Library General Public License, version 2, 
```

Comment:
Thanks. I'd always been a bit nervous about whether this counted as a licence change and hadn't ever managed to find a straight answer.

Reply:
I'm not a lawyer, but even if it is a license change it is just from LGPL to LGPL....
That being said the official text of LGPL v 2.1 does not feature the word General and TBH these files can probably just be ignored by spellintian since they aren't supposed to change and are "correct"....

(The main reason I did it was I was looking for a change that would cause travis to run again and hopefully pass this time)

Reply:
> I'm not a lawyer, but even if it is a license change it is just from LGPL to LGPL....

GNU LPL to GNU LGPL surely?

> That being said the official text of LGPL v 2.1 does not feature the word General and TBH these files can probably just be ignored by spellintian since they aren't supposed to change and are "correct"....

I'd rather only ignore stuff we can't change, not just stuff we choose not to.

> (The main reason I did it was I was looking for a change that would cause travis to run again and hopefully pass this time)

Don't worry about that. I can re-run them if necessary, and we've currently got bigger issues on OS X and With GCC8 anyway...

Reply:
> > I'm not a lawyer, but even if it is a license change it is just from LGPL to LGPL....
> 
> GNU LPL to GNU LGPL surely?
I looked up the language of LGPL and it is what we have.
https://www.gnu.org/licenses/old-licenses/lgpl-2.1.html
> 
> > That being said the official text of LGPL v 2.1 does not feature the word General and TBH these files can probably just be ignored by spellintian since they aren't supposed to change and are "correct"....
> 
> I'd rather only ignore stuff we can't change, not just stuff we choose not to.
Yeah will push an undo.
> 
> > (The main reason I did it was I was looking for a change that would cause travis to run again and hopefully pass this time)
> 
> Don't worry about that. I can re-run them if necessary, and we've currently got bigger issues on OS X and With GCC8 anyway...

Reply:
Thanks @Keeper-of-the-Keys . While I'd looked at the GNU website I'd not thought to check their version of the licence!